lms/djangoapps/grades/management/commands/backfill_grades.py

```diff
@@ -0,0 +1,141 @@
+"""
+Command to compute all grades for specified courses.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+
+from django.core.management.base import BaseCommand, CommandError
+import six
+
+from openedx.core.lib.command_utils import (
+    get_mutually_exclusive_required_option,
+    validate_dependent_option,
+    parse_course_keys,
+)
+from student.models import CourseEnrollment
+from xmodule.modulestore.django import modulestore
+
+from ... import tasks
+
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Example usage:
+        $ ./manage.py lms backfill_grades --all_courses --settings=devstack
+        $ ./manage.py lms backfill_grades 'edX/DemoX/Demo_Course' --settings=devstack
+    """
+    args = '<course_id course_id ...>'
+    help = 'Computes and persists grade values for all learners in specified courses.'
```

Comment:
Nit: Persisting of grades is specified by a different setting and not controlled by this command.  So I would remove the "and persists" part.